apps/api/v2/src/modules/organizations/interceptors/output-event-types-response.interceptor.ts

```diff
@@ -0,0 +1,33 @@
+import { EventTypeHandlerResponse } from "@/modules/organizations/controllers/event-types/organizations-event-types.controller";
+import { OutputOrganizationsEventTypesService } from "@/modules/organizations/services/event-types/output.service";
+import { CallHandler, ExecutionContext, Injectable, NestInterceptor } from "@nestjs/common";
+import { Observable } from "rxjs";
+import { switchMap } from "rxjs/operators";
+
+@Injectable()
+export class OutputEventTypesResponseInterceptor<T> implements NestInterceptor {
```

Comment:
I would rename it to `OutputTeamEventTypesResponseInterceptor` to distinguish it from user event type interceptor.